packages/flutter/lib/src/gestures/drag_details.dart

```diff
@@ -35,6 +40,15 @@ class DragDownDetails with Diagnosticable implements PositionedGestureDetails {
   @override
   final Offset localPosition;
 
+  /// The kind of the device that initiated the event.
+  final PointerDeviceKind? kind;
+
+  /// {@macro flutter.gestures.PointerEvent.buttons}
+  ///
+  /// NOTE: this will always be set by the platform but synthetic events might
```

Comment:
didn't feel great to hide a NOTE inside the macro (and it might not apply everywhere) so I left it here, but happy to pivot.

Reply:
would you still put that into a macro? it seems we are going in the direction of more customized per-event messages (such as referring to drag-specifics or referring to things with the exact name in the context they are). makes sense to just not use a macro at all?